README.md

```diff
@@ -2,98 +2,14 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/0e6a23344616e29b4ed0/maintainability)](https://codeclimate.com/github/SAP/jenkins-library/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/0e6a23344616e29b4ed0/test_coverage)](https://codeclimate.com/github/SAP/jenkins-library/test_coverage)
 
-# Description
+# Project Piper Library Repository
+Welcome to the Repository of the extended Library of Project Piper. 
```

Comment:
That's copied from Jenkins.io.

Reply:
Sounds better IMO.

But what is 

> the Jenkins Pipeline

IMO there is no "the Jenkins Pipeline". We have a pipeline script (normally named "Jenkinsfile" (but that is only a convention) in the project. But is that "the Jenkins pipeline"? I read it also as "THE Jenkins pipeline", as some kind of pre-defined build script in Jenkins.

Also 

> SAP specific pattern

does not tell anything to me. To some extend I'm afraid 😫 when reading this. 




I guess what should expressed here is something like: "We provide hand and fancy steps simplifying your life by hiding tool complexity so that you can focus on your business rather than fighting with all the build tools and technical details."

Reply:
'Jenkins Pipeline', see https://jenkins.io/doc/book/pipeline/, makes clear that this lib runs under Jenkins only. Furthermore, it is based on the 'Pipeline' of Jenkins.

"SAP patterns", makes clear that the lib contains SAP specific stuff.

"Shared Library" face "common patterns <which> are likely to emerge"